config/v1/types_cluster_version.go

```diff
@@ -88,6 +88,21 @@ type ClusterVersionSpec struct {
 	// +optional
 	Capabilities *ClusterVersionCapabilitiesSpec `json:"capabilities,omitempty"`
 
+	// SignatureStores contains the upstream URIs to verify release signatures.
```

Comment:
Convention says this should match the json tag
```suggestion
	// signatureStores contains the upstream URIs to verify release signatures.
```